README.md

```diff
@@ -1,29 +1,28 @@
-# IBM Cloud Satellite Module
+# IBM Cloud Satellite module
 
-Use this terrafrom automation to set up satellite location on IBM cloud.
-It will provision satellite location and create 6 VSIs and assign 3 host to control plane, and provision ROKS satellite cluster and auto assign 3 host to cluster,
-Configure cluster worker pool to an existing ROKS satellite cluster.
+Use this Terraform automation to set up a Satellite location on IBM cloud. The module provisions the IBM Cloud® Satellite location, creates 6 VSIs, assigns three hosts to the control plane, provisions a ROKS Satellite cluster, assigns three hosts to the cluster, and configures a cluster worker pool to an existing ROKS Satellite cluster.
```

Comment:
some inconsistency on "IBM Cloud" here and a little lower down.  First sentence, Cloud should be capitalized.  Second sentence, the trademark symbol is there, but it's removed down on line 14.